### DISCUSSION TITLE
Are there any plans to support JavaScript?

### DISCUSSION THREAD

**Discussion by serdar-** [marimo-team/marimo#596] [Ideas]
I really like Marimo, it is a really neat tool for Pluto.jl fans like me. I was wondering if you have plans to support running JavaScript inside the cells, similar to what Pluto.jl can do.

**Comment by akshayka** [comment]
We don't have plans for that yet, but it's something we could look into. Pluto's javascript runtime is neat for sure. Out of curiosity, do you have a use case in mind?

**Reply by serdar-** [reply]
Not a really specific use case, however, earlier I used  [echarts](https://echarts.apache.org/en/index.html), [3mol.js](https://3dmol.csb.pitt.edu/) and Canvas API with Pluto for example.
